### PR TITLE
Sims

### PR DESCRIPTION
**Pull request by pylat**

### CODE REVIEW

File: experiments/cubic_sparse_logreg/runme.jl [hunk 1]
```diff
@@ -66,61 +66,74 @@ function run_cubic_logreg_data(
...
 
+
+    x0 = zeros(n)
+    x_pert = x0 + randn(size(x0))
+
+    grad_x, _ = ProximalCore.gradient(f, x0)
+    grad_x_pert, _ = ProximalCore.gradient(f, x_pert)
+    gam_init = norm(x0 - x_pert)^2 / dot(grad_x - grad_x_pert, x0 - x_pert) 
+
+    println(gam_init)
```
Comment (lostella) [inline experiments/cubic_sparse_logreg/runme.jl]: we keep this here?

**Comment by lostella** [comment]
can you remove the pdf files?

File: experiments/cubic_sparse_logreg/runme.jl (hunk 1 above)
Comment (pylat) [reply experiments/cubic_sparse_logreg/runme.jl]: I guess, we decided to initialize like this. It was better than just randomly setting gamma = 1

Comment (lostella) [reply experiments/cubic_sparse_logreg/runme.jl]: I mean the `println`

Comment (pylat) [reply experiments/cubic_sparse_logreg/runme.jl]: updated it

**Review by lostella — APPROVED** [review]

**Pull request merged**